README.md

```diff
@@ -45,6 +45,7 @@ If you want to override that, specify an `CODE_GOV_API_KEY` environmental variab
 ```
 CODE_GOV_API_KEY=l87sfdi7ybc2bic7bai8cb2i176c3b872tb3 npm run start
 ```
+An alternate approach to using your API key every time you use ```npm run start``` is to create a ```.env.local``` file and store your API key. (Remember to use ```CODE_GOV_API_KEY=...```) In accordance with [dotenv-flow](https://www.npmjs.com/package/dotenv-flow), your personal key will be ignored when committing updates to GH.
```

Comment:
You can just use single backticks to surround single code-block items. The triple is used for multi-line code blocks.

Reply:
Ok, thanks!